index.html

```diff
@@ -1876,6 +1876,85 @@ <h3>Examples</h3>
       </pre>
     </section>
   </section>
+  <section>
+    <h2>Background Blur Effect Status</h2>
+    <p>The {{VideoFrameMetadata}} interface exposes the blur state as a property,
+      which allows apps to know the state for every frame. This is important for
+      scenarios where the app must ensure user privacy by never sending an
```

Comment:
Let's avoid normative "must". I also don't think we want to overstate the security properties of camera blurring. OS implementations might differ significantly, and they don't necessarily pass everyone's definition of a security measure.

```suggestion
      which allows apps to know the state for every frame. This can be helpful for
      scenarios where the app wishes to help protect user privacy by never sending an
```